test/integration/model/sync.test.js

```diff
@@ -215,12 +215,14 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           const results = await this.sequelize.getQueryInterface().showIndex(User.getTableName());
           if (dialect === 'sqlite') {
             // SQLite doesn't treat primary key as index
-            expect(results).to.have.length(4);
+            // However it does create an extra "autoindex", except primary == false
+            expect(results).to.have.length(4 + 1);
           } else {
             expect(results).to.have.length(4 + 1);
             expect(results.filter(r => r.primary)).to.have.length(1);
           }
 
+          expect(results.filter(r => r.name === 'sqlite_autoindex_testSyncs_1')).to.have.length(1);
```

Comment:
You are talking about this unique index, right? It can be wrapped with `if (dialect === 'sqlite')` statement